package/libyui.spec

```diff
@@ -16,130 +17,94 @@
 #
 
 Name:           libyui
-Version:        3.12.2
-Release:        0
-Source:         %{name}-%{version}.tar.bz2
 
-%define so_version 14
-%define bin_name %{name}%{so_version}
+# DO NOT manually bump the version here; instead, use   rake version:bump
```

Comment:
*rake version:bump* only increases the patch number, right? This comment might be confusing if we need to bump major/minor version.

Reply:
For one thing, that is a shortcoming of that `rake version:bump`; for another, this comment is meant as a reminder that those libyui packages need a more special handling for bumping version numbers. It happened in the past that people changed only one of those 3 places (the toplevel VERSION.cmake and the two .spec files).

BTW we had this comment in some, but not all of our libyui*.spec files; now they all have it.